pykwalify/core.py

```diff
@@ -486,16 +486,17 @@ def _validate_mapping(self, value, rule, path, errors, done=None):
                         sub_regex_result.append(False)
 
                 if rule._matching_rule == "any":
+
                     if any(sub_regex_result):
-                        log.debug(u" + Matched atleast one regex")
+                        log.debug(u" + Matched at least one regex")
                     else:
                         log.debug(u"No regex matched")
                         errors.append(SchemaError.SchemaErrorEntry(
                             msg=u"Key '{key}' does not match any regex '{regex}'. Path: '{path}'",
                             path=path,
-                            value=value.encode('unicode_escape'),
+                            value=value,
                             key=k,
-                            regex="  ".join([mm[0]._map_regex_rule for mm in regex_mappings])))
+                            regex="' or '".join(sorted([mm[0]._map_regex_rule for mm in regex_mappings]))))
```

Comment:
@Grokzen I believe sorted method on the list comprehension has made the error message more predictable.   Local tests and CI reports good results.